### PR TITLE
[HTP-0004] Add ccp support, correct versioning, lint fix

### PR DESCRIPTION
**Pull request by HuddledMasses**
## Type of Change

- [ ] New adapter
- [x] Feature Update
- [ ] Bug Fix
- [ ] Code Refactor
- [ ] Other

### Related Issue

#

### CODE REVIEW

File: colossus/colossus-htb.js [hunk 1]
```diff
@@ -175,9 +175,12 @@ function ColossusHtb(configs) {
...
          */
-        queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
-        queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
-
+        if (ComplianceService) {
+            queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
+            queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
+            queryObj.ccpa = ComplianceService.usp && ComplianceService.usp.getConsent();
+        }
+
```
Comment (ix-certification) [inline colossus/colossus-htb.js]: Hi,

Lint test is still failing on this line.
`183:1  error  Trailing spaces not allowed  no-trailing-spaces`
Can you please remove the trailing white spaces?

Thanks!

Comment (HuddledMasses) [reply colossus/colossus-htb.js]: done

File: colossus/colossus-htb.js [hunk 2]
```diff
@@ -175,9 +175,11 @@ function ColossusHtb(configs) {
...
          * made by the wrapper to contact a Consent Management Platform.
          */
-        queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
-        queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
-
+        if (ComplianceService) {
+            queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
+            queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
+            queryObj.ccpa = ComplianceService.usp && ComplianceService.usp.getConsent();
+        }
```
Comment (ix-certification) [inline colossus/colossus-htb.js]: Hi,
Thanks for fixing the previous linting error.
However, there seem to be 2 new ones created in the process.

`182:10  error  Trailing spaces not allowed   no-trailing-spaces`
`183:9   error  Expected line before comment  lines-around-comment`

You can run the lint test as instructed in the link below.
https://kb.indexexchange.com/Adapters/ESLint.htm

Thanks!

Comment (HuddledMasses) [reply colossus/colossus-htb.js]: Done

File: just-premium/just-premium-htb-validator.js [hunk 3]
```diff
@@ -10,7 +10,6 @@
  * prior written permission of Index Exchange.
  */
 
-
```
Comment (ix-certification) [inline just-premium/just-premium-htb-validator.js]: Hi!

Thanks for fixing the lint but can you please remove the changes done on the just-premium files?
Just leave the changes done for `colossus/CHANGES.md` and `colossus/colossus-htb.js`. 

Thanks

**Comment by HuddledMasses** [comment]
Done

**Comment by HuddledMasses** [comment]
I've reverted previous commited and made changes n colossus-htb only

**Comment by ix-certification** [comment]
Hi!
Looks great!
Just so your team is aware, we will be passing an empty string if for whatever reason we are not able to pass back a consent. Reasons such as CCPA times out or the API is down.

Cheers!

**Pull request merged**